core-api/src/main/java/com/optimizely/ab/event/NoopEventHandler.java

```diff
@@ -29,7 +29,8 @@
     private static final Logger logger = LoggerFactory.getLogger(NoopEventHandler.class);
 
     @Override
-    public void dispatchEvent(String url, Map<String, String> params) {
-        logger.debug("Called dispatchEvent with URL: {} and params: {}", url, params);
+    public void dispatchEvent(LogEvent logEvent) {
```

Comment:
Passing a `Map<String, String>` makes it so that you must perform explicit serialization rather than using an existing library like Jackson or Gson (can't perform type inference without POJOs), so we'll have typecasts from `String` for each POST parameter. This isn't an issue with untyped languages since a dictionary can store multiple types, but in Java it's difficult to generate a payload string from a `Map` and the convention is to serialize from a POJO.  I think using `LogEvent` has a low learning curve since it doesn't contain any Optimizely-specific logic.